addons/website/static/src/scss/website.scss

```
@@ -1378,6 +1378,36 @@ header {
     }
 }
 
+// Step wizard used in /shop, /event, appointment
+.o_wizard {
+    $o_wizard_circle_progress_size: 64px;
+
+    .o_wizard_step-active {
```

Frontend convention: custom classes' words separator is always lowercase
 ```suggestion
    .o_wizard_step_active {
```